docs/src/main/asciidoc/se/webclient.adoc

```diff
@@ -407,29 +407,56 @@ include::{sourcedir}/se/WebClientSnippets.java[tag=snippet_8,indent=0]
 
 === Adding Service to WebClient
 
-WebClient currently supports 3 built-in services namely `metrics`, `tracing` and `security`.
+WebClient currently supports four built-in services, namely
+xref:discovery.adoc#_web_client_discovery_integration[`discovery`], `metrics`, `tracing` and `security`.
 
 ==== Enabling the service
 
-In order for a service to function, their dependency needs to be added in the application's pom.xml. Below are examples on how to enable the built-in services:
+In order for a service to function, their dependency needs to be added in the application's `pom.xml`. Below are examples on how to enable the built-in services:
```

Comment:
As long as we're editing this line, maybe also change "their dependency" to "its dependency" or "its dependencies" to match the antecedent "a service".

Reply:
Addressed in https://github.com/helidon-io/helidon/pull/10786/commits/1e5d5661d3cff09e0a74238b9c186787caf25e8e